web_ir_actions_act_view_reload/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+crud_ir_actions_act_view_reload,access_ir_actions_act_wiew_reload,model_ir_actions_act_view_reload,,1,1,1,1
```

Comment:
Do the user need write, delete, create rights?

Reply:
I copied https://github.com/OCA/web/blob/14.0/web_ir_actions_act_multi/security/ir.model.access.csv
and https://github.com/OCA/web/blob/14.0/web_ir_actions_act_window_message/security/ir.model.access.csv

Reply:
I agree with tarteo... look at Odoo: https://github.com/odoo/odoo/blob/14.0/odoo/addons/base/security/ir.model.access.csv#L85

Reply:
Yeah, the main difference is the group :) ... No Group = All users can do the operations.

Reply:
@Tardo, aaaah I missed that

Reply:
so, do you want me to add the group like in base Odoo? Maybe on all three modules?

Reply:
Yes, i think that this change should be made on all modules. Thanks!